h2o-automl/src/main/java/ai/h2o/automl/modeling/StackedEnsembleStepsProvider.java

```diff
@@ -115,6 +116,9 @@ protected StackedEnsembleParameters getStackedEnsembleParameters(Key<Model>[] ba
                 stackedEnsembleParameters.initMetalearnerParams();
                 stackedEnsembleParameters._metalearner_parameters._keep_cross_validation_models = buildSpec.build_control.keep_cross_validation_models;
                 stackedEnsembleParameters._metalearner_parameters._keep_cross_validation_predictions = buildSpec.build_control.keep_cross_validation_predictions;
+                // add custom alpha in GLM metalearner
+                GLMModel.GLMParameters metalearner_params = (GLMModel.GLMParameters)stackedEnsembleParameters._metalearner_parameters;
+                metalearner_params._alpha = new double[]{0.5, 1.0};
```

Comment:
do we add it only for AutoML or shouldn't we rather have it for the AUTO metalearner?
wdyt? Also documentation-wise.

Reply:
my main concern here is that we introduce 2 levels of AUTO that may confuse the user: the AUTO SE and the AUTO SE in AUTO ML

Reply:
My old PR had it for default SE, but I thought it was best to keep this PR isolated & simple.  I do think we could consider updating the default metalearner in SE as well, but I didn't have time to fully think about the repercussions for doing that (e.g. reproducibility, tests, etc), and also it's harder to do that neatly on the client-side since the alpha array is only supported on the backend.